python/searchSystemPackageHardware.py

```diff
@@ -0,0 +1,33 @@
+# list the hardware devices in a system package
+# ex: python3 searchSystemPackageHardware.py http://192.168.13.111:8080 companyinfra openrmfprosvc hvs.xxxxxxxxxxxxxx
```

Comment:
The example up top here is missing that 5th argument. An example would be the below.

```
# ex: python3 searchSystemPackageHardware.py http://192.168.13.111:8080 companyinfra openrmfprosvc hvs.xxxxxxxxxxxxxx "red hat"
```